src/bio/bio_context.c

```diff
@@ -672,7 +717,7 @@ __bio_ioctxt_open(struct bio_io_context **pctxt, struct bio_xs_context *xs_ctxt,
 	}
 
 	ctxt->bic_xs_blobstore = bxb;
-	rc = bio_blob_open(ctxt, false);
+	rc = bio_blob_open(ctxt, false, flags & BIO_MC_FL_SYSDB);
```

Comment:
So the super blob could be opened twice? One for meta blob open, the other for WAL blob open? I'm wondering if I should change the WAL design a bit to support Meta & WAL on same blob, so that we could put meta & WAL blob for sysdb on the super blob.